setup.py

```diff
@@ -116,6 +116,7 @@ def run(self):
     },
     install_requires=[
         'sphinx'
+        'docutils<0.17', # https://github.com/sphinx-doc/sphinx/issues/9001
```

Comment:
Missed the preceding `,` in the list here. Requires got parsed as `sphinxdocutils`